index.html

```diff
@@ -0,0 +1,304 @@
+                  <p class="role">Back End Dev</p>
+                  <span class="separator-circle"></span>
+                  <p class="year">2015</p>
+                </div>
+              </div>
+              <p class="project-details">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tags">
+                <li class="tag">html</li>
+                <li class="tag">css</li>
+                <li class="tag">javaScript</li>
+              </ul>
+              <div class="action">
+                <button class="btn-default" type="button">See Project</button>
+              </div>
+            </div>
+          </article>
+        </section>
+        <section class="about bg-white">
+          <h2 class="title">About Myself</h2>
+          <p class="details">
+            I’m a software developer! If you have a project you need coded, do not hestiate to
+            contact me.
+          </p>
+          <div class="social-media">
+            <a class="contact-link" href="#">LET’S CONNECT</a>
+            <ul class="social-media-links">
+              <li>
+                <a href="#"
+                  ><img src="image/tw.png" alt="Twitter"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    src="image/linkedin.png"
+                    alt="LinkedIn"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/medium.png" alt="Medium"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/github.png" alt="Github"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="image/v.png" alt="V"
+                /></a>
+              </li>
+            </ul>
+            <button class="btn-default" type="button">Get my bio-data</button>
+          </div>
+          <ul class="skill-list">
+            <li class="my-skill">
+              <div class="skill-container">
+                <h3 class="skill-title">Languages</h3>
+                <button class="btn-chev" type="button">
+                  <img src="images/chev-dow.svg" alt="" />
+                </button>
+              </div>
+              <!-- <div class="separator-line"></div> -->
+              <ul class="language-list">
+                <li>
+                  <img src="images/javascript.svg" alt="JavaScript" />
+                  JavaScript
+                </li>
+                <li>
+                  <img src="images/html.svg" alt="HTML" />
+                  HTML
+                </li>
+                <li>
+                  <img src="images/css.svg" alt="CSS" />
+                  CSS
+                </li>
+              </ul>
+            </li>
```

Comment:
- [ ] **[REQUIRED]**  As the previous reviewer mentioned, please fix the language icons image and  stick to the design as much as possible because as a developer it is very important peer attention to detail, your future employer will judge your work :+1:

Reply:
I did it